docs/features/pytorch.md

```diff
@@ -0,0 +1,140 @@
+- [torchvision](https://prefix.dev/channels/conda-forge/packages/torchvision)
+- [torchaudio](https://prefix.dev/channels/conda-forge/packages/torchaudio)
+- [cuda-version](https://prefix.dev/channels/conda-forge/packages/cuda-version)
+
+## Installing from PyPi
+Thanks to the integration with `uv` we can also install PyTorch from `pypi`.
+
+### Pytorch index
+PyTorch packages are provided through a custom index, which is maintained by the PyTorch team.
+To install PyTorch from the PyTorch index, you need to add the indexes to manifest.
+Best to do this per dependency to force the index to be used.
+
+- CPU only: [https://download.pytorch.org/whl/cpu](https://download.pytorch.org/whl/cpu)
+- CUDA 11.8: [https://download.pytorch.org/whl/cu118](https://download.pytorch.org/whl/cu118)
+- CUDA 12.1: [https://download.pytorch.org/whl/cu121](https://download.pytorch.org/whl/cu121)
+- CUDA 12.4: [https://download.pytorch.org/whl/cu124](https://download.pytorch.org/whl/cu124)
+- ROCm6: [https://download.pytorch.org/whl/rocm6.2](https://download.pytorch.org/whl/rocm6.2)
+
+=== "`pixi.toml`"
+    ```toml title="Install PyTorch from pypi"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-pypi.toml:minimal"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Install PyTorch from pypi"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-pypi.toml:minimal"
+    ```
+
+## Installing from PyTorch channel
+!!! warning
+    This depends on the [non-free](https://www.anaconda.com/blog/is-conda-free) `main` channel from Anaconda and mixing it with `conda-forge` can lead to conflicts.
+
+!!! note
+    This is the [legacy](https://dev-discuss.pytorch.org/t/pytorch-deprecation-of-conda-nightly-builds/2590) way of installing pytorch, this will not be updated to later versions as pytorch has discontinued their channel.
+
+=== "`pixi.toml`"
+    ```toml title="Install PyTorch from the PyTorch channel"
+    --8<-- "docs/source_files/pixi_tomls/pytorch-from-pytorch-channel.toml:minimal"
+    ```
+=== "`pyproject.toml`"
+    ```toml title="Install PyTorch from the PyTorch channel"
+    --8<-- "docs/source_files/pyproject_tomls/pytorch-from-pytorch-channel.toml:minimal"
+    ```
+
+## Troubleshooting
+
+- You can test the installation with the following command:
+```shell
+pixi run python -c "import torch; print(torch.__version__); print(torch.cuda.is_available())"
+```
+- You can ask pixi which version of CUDA it finds on your computer with `pixi info`.
+```
+> pixi info
+...
+Virtual packages: __unix=0=0
+                : __linux=6.5.9=0
+                : __cuda=12.5=0
+...
+```
+- Installing `torch` from PyPI and other packages from Conda channels doesn't work.
+  - The lowest level package in the dependency tree that uses a PyPI package demands that all later packages are also PyPI packages.
+- Reasons for broken installations
+  - Using both `conda-forge` and `pytorch` channels, this can lead to conflicts. Choose one or the other.
```

Comment:
Maybe mention again that the `pytorch` channel is deprecated?

Reply:
I don't think the `pytorch` channel is deprecated.  However, mixing it with `conda-forge` is unsupported as it is supposed to be installed alongside `defaults` and `nvidia`.  In general, guaranteeing compatibility between packages compiled with different build-time dependencies requires careful checking.  I think that the way it is written is OK as it emphasises mixing different compiled software stacks can lead to trouble.

Reply:
This is now written differently.